packages/manager/modules/vrack/src/dashboard/vrack.service.js

```diff
@@ -17,4 +17,34 @@ export default class Vrack {
   getIpInfo(ip) {
     return this.$http.get(`/ip/${encodeURIComponent(ip)}`);
   }
+
+  getEligibleServices(serviceName) {
+    return this.$http.get(`/vrack/${serviceName}/eligibleServices`);
+  }
+
+  getServiceInformation(url, serviceName) {
+    return this.$http.get(`${url}/${serviceName}/serviceInfos`);
+  }
+
+  getDedicatedServer(serviceName) {
+    return this.$http.get(`/dedicated/server/${serviceName}`);
+  }
+
+  getDedicatedServerInterface(dedicatedServerInterface) {
+    return this.$http.get(
+      `/dedicated/server/${dedicatedServerInterface.dedicatedServer}`,
+    );
+  }
```

Comment:
Why duplicate the function here, it's not necessary, you should just call the function with the correct parameter
```suggestion
  getDedicatedServer(serviceName) {
    return this.$http.get(`/dedicated/server/${serviceName}`);
  }

```
```
VrackService.getDedicatedServer(dedicatedServerInterface.dedicatedServer)
```